cmd/devices-api/main.go

```diff
@@ -226,7 +230,20 @@ func startTaskStatusConsumer(logger zerolog.Logger, settings *config.Settings, p
 
 	ddSvc := services.NewDeviceDefinitionService(pdb.DBS, &logger, settings)
 
-	taskStatusService := services.NewTaskStatusListener(pdb.DBS, &logger, ddSvc, kp, settings)
+	accountsConn, err := grpc.NewClient(settings.AccountsAPIGRPCAddr, grpc.WithTransportCredentials(insecure.NewCredentials()))
+	if err != nil {
+		logger.Fatal().Err(err).Msg("Failed to create users API client.")
+	}
+	defer accountsConn.Close()
+
+	acctClient := pb_accounts.NewAccountsClient(accountsConn)
+
+	cioSvc, err := cio.New(settings.CustomerIOAPIKey, acctClient, &logger)
+	if err != nil {
+		logger.Fatal().Err(err).Msg("failed to start customer io service")
```

Comment:
```suggestion
		logger.Fatal().Err(err).Msg("failed to create customer io service")
```
This is picky but it's not, like, a daemon in my view.